src/store/BouquetStore.js

```diff
@@ -29,12 +29,15 @@ export const useBouquetStore = defineStore("bouquet", {
      */
     async loadBouquets () {
       if (this.data.length > 0) return this.data
-      let response = await topicsAPI._list()
-      this.data = this.filter(response.data)
+      const headers = {"X-Fields": "data{id,slug,tags},next_page"}
+      let response = await topicsAPI._list(headers)
+      let bouquets = this.filter(response.data)
+      // FIXME: Should this logic be included in `_list`? Maybe with an optional `paginate` param?
       while (response.next_page) {
-        response = await topicsAPI.request(response.next_page)
-        this.data = [...this.data, ...this.filter(response.data)]
+        response = await topicsAPI.request(response.next_page, "get", {"headers": headers})
+        bouquets = [...bouquets, ...this.filter(response.data)]
       }
+      this.data = await Promise.all(bouquets.map(b => topicsAPI._get(b.id)))
```

Comment:
Je pense qu'ici tu pourrais profiter du caractère _réactif_ du front. Par example, laisser plutôt le store itérer. Cela ne change pas le problème que tu essaies de résoudre, mais toute de même permet d'avoir un rendereing progressif du côté de l'utilisateur. (Si c'est n'est déjà le cas).

Reply:
Pour m'instruire, tu ferais ça comment ?

Reply:
Je laisse @YeLnatSs répondre 😄 . Mais le principe est que qu'on créer un _bind_ entre la UI et un _state_ (le store par example). Donc, au fur et à mesure qu'on pousse de la donnée en _chunks_ au store, la UI devrait normalement s'actualiser progressivement.